SyncConnectionStateErrorReset/SyncConnectionStateErrorReset/SyncConnectionStateErrorResetView.swift

```diff
@@ -0,0 +1,261 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2023 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+import SwiftUI
+import RealmSwift
+
+// Entry Point
+struct InitialView: View {
+    @ObservedObject var realmManager = RealmManager()
+    @ObservedObject var app: RealmSwift.App
+    @State var isLogged: Bool = false
```

Comment:
I'm a little bit confused about this `isLogged` variable. It looks like we're passing it in to the `LoginView`, so the `LoginView` will get its initial value from here - but the LoginView _also_ has a `@State` variable. Was that meant to be a `@Binding` variable to pass its value back up to here? Otherwise, I'm not sure why we're bothering with initializing this here if we're only using it in `LoginView` - why not initialize it there?

Also, we might want to adjust the naming. The `isLogged` name makes me think we're talking about logging, but I think we're referring to login state? If so, we might want this to be `isLoggedIn`.

Reply:
Removed